README.md

```diff
@@ -75,8 +75,8 @@ The error object can be passed along with the original source to `Parsimmon.form
     any character except one of the characters `"a"`, `"b"`, or `"c"`,
     and will yield the same.
   - `Parsimmon.regex(/myregex/, group=0)` is a parser that expects the stream
-    to match the given regex, and yields the given match group, or the
-    entire match.
+    to match the given regexp, and yields the given match group, or the
+    entire match. The regexp may only use the following flags: `imu`. `Parsimmon.regex` will throw an error for any other flag. The regexp is also wrapped in `^(?: regexp )` because it doesn't make sense to match something further away than the next character. Greedy regexp should be wrapped like this: `( regexp )+` (e.g. `Parsimmon.regex(/(a+|a*b)+/)`).
```

Comment:
> matching begins at the parser's cursor
I like the way that sounds better.

> We don't need a regexp basics tutorial to document this parser
I'm just going off @Risto-Stevcev 's suggestion related to their issue they filed, but tbh I never really understood what they were getting at.